README.md

```diff
@@ -77,6 +77,24 @@ bin/elixirc lib/elixir/lib/string.ex -o lib/elixir/ebin
 bin/elixir lib/elixir/test/elixir/string_test.exs
 ```
 
+To recompile only changed files (including Erlang modules):
```

Comment:
@samphilipd if you change a single file in the elixir application, the whole application will be recompiled. The same for mix, eex, etc.